tests/api/test_feeds.py

```diff
@@ -61,10 +70,23 @@ def test_order(self, client, order):
         assert len(feed.xpath('//atom:entry', namespaces=NAMESPACES)) == 11
 
         for creative_work, entry in zip(works, feed.xpath('//atom:entry', namespaces={'atom': 'http://www.w3.org/2005/Atom'})):
+            try:
+                contributors = AgentWorkRelation.objects.filter(creative_work_id=creative_work.id)
```

Comment:
This queryset is evaluated up to three different times, I think it'd be better to wrap it in `list()` here.

Also, I think if you use `AbstractAgentWorkRelation`, it'll fix your `order_cited` problems (because yay typedmodels!)